Kitodo/src/main/java/org/kitodo/model/LazyDTOModel.java

```diff
@@ -56,27 +52,6 @@ public LazyDTOModel(SearchService searchService) {
         }
     }
 
-    /**
-     * Creates a LazyDTOModel instance that allows fetching data from the data
-     * source lazily, e.g. only the number of datasets that will be displayed in the
-     * frontend.
-     *
-     * @param additionalFiltering true, set up only in ProzessverwaltungForm or SearchForm
-     * @param searchService
-     *            the searchService which is used to retrieve data from the data
-     *            source
-     */
-    public LazyDTOModel(boolean additionalFiltering, SearchService searchService) {
-        this.additionalFiltering = additionalFiltering;
-        this.searchService = searchService;
-        try {
-            this.setRowCount(toIntExact(searchService.count()));
```

Comment:
It shouldn't be removed as it is more likely needed for displaying amount of results.